### PR TITLE
Add guide command

### LINKED ISSUE #11: new command: guide
### Expected behavior

Executing the `lingo guide` command opens up a page in the default browser that describes all the rules applicable for the current project. It contains the names of the rules as well as good and bad examples of applying them.

### PR DESCRIPTION
**Pull request by s2gatev**
Closes #11

### CODE REVIEW

File: checker/checker.go [hunk 1]
```diff
@@ -23,9 +23,28 @@ type Report struct {
 	Errors []Error
 }
 
+// Example shows how to adhere and not adere to a rule.
+type Example struct {
+
+	// Good is an example of sticking to the rule.
+	Good string
+
+	// Bad is a counter-example that shows how to not apply the rule.
```
Comment (catiepg) [inline checker/checker.go]: How about `Bad is a counter-example showing a mis-use which lingo will report. ` or something along these lines?

File: checker/pass_context_first_checker.go [hunk 2]
```diff
@@ -18,6 +18,26 @@ func NewPassContextFirstChecker(configData interface{}) NodeChecker {
...
 }
 
+// Title implements the NodeChecker interface.
+func (c *PassContextFirstChecker) Title() string {
+	return "Context Argument First"
+}
+
+// Description implements the NodeChecker interface.
+func (c *PassContextFirstChecker) Description() string {
+	return `A function must receive context.Context as its first argument.`
```
Comment (catiepg) [inline checker/pass_context_first_checker.go]: How about `When a function receives context.Context it must be its first argument.`?

File: checker/return_error_last_checker.go [hunk 3]
```diff
@@ -18,6 +18,26 @@ func NewReturnErrorLastChecker(configData interface{}) NodeChecker {
 	return &ReturnErrorLastChecker{}
 }
 
+// Title implements the NodeChecker interface.
+func (c *ReturnErrorLastChecker) Title() string {
+	return "Error Return Last"
```
Comment (catiepg) [inline checker/return_error_last_checker.go]: It seems to me that `Return Error Last` is clearer.

File: checker/return_error_last_checker.go [hunk 4]
```diff
@@ -18,6 +18,26 @@ func NewReturnErrorLastChecker(configData interface{}) NodeChecker {
...
+}
+
+// Description implements the NodeChecker interface.
+func (c *ReturnErrorLastChecker) Description() string {
+	return `A function must return error as its last argument.`
```
Comment (catiepg) [inline checker/return_error_last_checker.go]: Similar to the `context.Context` checker, I suggest `When a function returns error it must be its last return value.`.

File: cmd/guide.go [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+		}
+
+		guide, err := os.Create(filepath.Join(dir, "guide.html"))
+		if err != nil {
+			// TODO: handle error gracefully
+			panic(err)
+		}
+		defer guide.Close()
+
+		code := `package main
```
Comment (catiepg) [inline cmd/guide.go]: Is this a leftover?

File: checker/checker.go (hunk 1 above)
Comment (s2gatev) [reply checker/checker.go]: Done.

File: checker/pass_context_first_checker.go (hunk 2 above)
Comment (s2gatev) [reply checker/pass_context_first_checker.go]: Done.

File: checker/return_error_last_checker.go (hunk 3 above)
Comment (s2gatev) [reply checker/return_error_last_checker.go]: Done.

File: checker/return_error_last_checker.go (hunk 4 above)
Comment (s2gatev) [reply checker/return_error_last_checker.go]: Done.

File: cmd/guide.go (hunk 5 above)
Comment (s2gatev) [reply cmd/guide.go]: Removed.

**Pull request merged**